### ISSUE TITLE
issue templateの作成

### ISSUE THREAD

**Issue by YukiYuigishi** [Undecided-Discord/Oshavery-Front#280]
TiDDをやる事が面倒くさくならないようにissue templateを作成します

現状考えているのはこの3つです。他に追加してほしいものがある人はコメントしてください

- [ ] 新機能
- [ ] bug報告
- [ ] 議論

**Comment by laminne** [comment]
"議論"、IssueじゃなくてDiscussionsを使うべきでは？
Issueが増えすぎるのはよろしく無いので(管理コスト的に

**Comment by YukiYuigishi** [comment]
管理コスト的に考えたらそうですね
議論はDiscussionを使いますか

**Comment by laminne** [comment]
> 管理コスト的に考えたらそうですね 議論はDiscussionを使いますか

そうですね, [Growthers/Oshavery](https://github.com/growthers/oshavery)にDiscussionsを置いて起きましょう

**Comment by YukiYuigishi** [comment]
dev もしくは masterにPRを投げるとき用のPull Request テーンプレートも作っといたほうが良さそうやね(できたっけ？

**Comment by laminne** [comment]
選択肢の一つとしては作れる

**Comment by YukiYuigishi** [comment]
あった方が手間を省けるから作るべきかなぁ

**Comment by laminne** [comment]
あってもいい気はする(すごく
